### PR TITLE
Admin Guide

### PR DESCRIPTION
**Pull request by daonb**
A one page guide based on common scenario. Included in the PR are some commits that have to do with code cleanup, scripts option's renaming, etc.

### CODE REVIEW

File: scripts/migrate.py [hunk 1]
```diff
@@ -323,13 +325,17 @@ def get_file_descriptors(tree, gedcom_path):
     return file_id, file_name
 
 
-def migrate_trees(cursor, treenum=None, gedcom_path=None, on_save=None, dryrun=False):
+def migrate_trees(cursor, on_save=None, treenum=None, gedcom_path=None):
```
Comment (OriHoch) [inline scripts/migrate.py]: why was dryrun parameter removed?

File: scripts/migrate.py [hunk 2]
```diff
@@ -339,11 +345,12 @@ def migrate_trees(cursor, treenum=None, gedcom_path=None, on_save=None, dryrun=F
             continue
 
         try:
-            g = Gedcom(fd=gedcom_fd)
+            gedcom = Gedcom(fd=gedcom_fd)
```
Comment (OriHoch) [inline scripts/migrate.py]: the gedcom object is referred to as `g` below - so should change there as well

File: scripts/migrate.py [hunk 3]
```diff
@@ -354,12 +361,12 @@ def migrate_trees(cursor, treenum=None, gedcom_path=None, on_save=None, dryrun=F
         logger.info('<<< migrated tree {}, path {}'
                     .format(row['GenTreeNumber'], file_name))
         count += 1
+
     return count
 
 
 if __name__ == '__main__':
     args = parse_args()
-    until = int(args.until)
```
Comment (OriHoch) [inline scripts/migrate.py]: `until` variable is still referenced in the code below - looks like it will cause error

File: scripts/migrate.py [hunk 4]
```diff
@@ -379,18 +386,19 @@ def migrate_trees(cursor, treenum=None, gedcom_path=None, on_save=None, dryrun=F
     else:
         since = int(args.since)
 
+    args.since = since
```
Comment (OriHoch) [inline scripts/migrate.py]: why? the `since` variable is referenced below

File: tests/test_migration.py [hunk 5]
```diff
@@ -65,7 +65,10 @@ def test_updated_doc(mocker, app):
             id=id,
             index = 'db',
         )
+        # reset ES mock
         elasticsearch.Elasticsearch.index.reset_mock()
+        mocker.patch('elasticsearch.Elasticsearch.index')
```
Comment (OriHoch) [inline tests/test_migration.py]: why does elasticsearch need to be mocked again? it is mocked above

**Comment by daonb** [comment]
@OriHoch took your code, from my changes, kept only the comment of `migrate_trees`

**Pull request merged**